depends/patches/intel-ipsec-mb/remove_digest_init.patch

```diff
@@ -0,0 +1,13 @@
+diff -dur a/lib/include/sha_generic.h b/lib/include/sha_generic.h
+index 3752546..77efd91 100644
+--- a/lib/include/sha_generic.h
++++ b/lib/include/sha_generic.h
+@@ -308,7 +308,7 @@ void sha_generic_1block(const void *data, void *digest,
+         if (data == NULL || digest == NULL)
+                 return;
+ #endif
+-        sha_generic_init(digest, sha_type);
++//        sha_generic_init(digest, sha_type);
```

Comment:
What's the source and purpose of this patch? The commit might be clearer if we can tell why it's here and under what circumstances we might want to remove the patch (if the library changes, for example).

Reply:
Thanks, here's the related discussion.. https://github.com/intel/intel-ipsec-mb/issues/86